datalad/distribution/create_sibling.py

```diff
@@ -102,7 +103,7 @@ def _create_dataset_sibling(
         # TODO: Is this condition valid for != '.' only?
         path_exists = True
         try:
-            out, err = ssh(["ls", remoteds_path])
+            out, err = ssh("ls {}".format(sh_quote(remoteds_path)))
```

Comment:
Couldn't we just sh_quote every item in the list and join it, while maintaining allowing for lists as an argument?